### PR TITLE
Display error when --path is invalid

### LINKED ISSUE #122: No error when --path is invalid
When passing `--path foobar` to `pack build`, no error is generated, and buildpacks simply fail detection.

### PR DESCRIPTION
**Pull request by jromero**
Resolves #122

Signed-off-by: Javier Romero <j.romero.1214@gmail.com>

**Output:**
image

image

### CODE REVIEW

File: build.go [hunk 1]
```diff
@@ -191,45 +201,6 @@ func (bf *BuildFactory) BuildConfigFromFlags(ctx context.Context, f *BuildFlags)
 	return b, nil
 }
 
-func Build(ctx context.Context, outWriter, errWriter io.Writer, appDir, builderImage, runImage, repoName string, publish, clearCache bool) error {
```
Comment (jromero) [inline build.go]: I took the liberty of removing this as it seemed unused and duplicate of what the command does.

File: run.go [hunk 2]
```diff
@@ -46,45 +42,6 @@ func (bf *BuildFactory) RunConfigFromFlags(ctx context.Context, f *RunFlags) (*R
 	return rc, nil
 }
 
-func Run(ctx context.Context, outWriter, errWriter io.Writer, appDir, buildImage, runImage string, ports []string) error {
```
Comment (jromero) [inline run.go]: I took the liberty of removing this as it seemed unused and duplicate of what the command does. Can revert if I was a bit hasty.

Comment (jromero) [reply run.go]: Reverted as I just realized this is intended to be used as a library as well.

File: build.go (hunk 1 above)
Comment (jromero) [reply build.go]: Reverted and adapted as I just realized this is intended to be used as a library as well.

**Comment by ekcasey** [comment]
@jromero Thanks for the contribution!

So far, we have intentionally avoided abstracting the filesystem, instead creating tmp dirs or using test fixtures when necessary for testing. I would like to stick to that pattern if possible.

**Comment by jromero** [comment]
> @jromero Thanks for the contribution!
> 
> So far, we have intentionally avoided abstracting the filesystem, instead creating tmp dirs or using test fixtures when necessary for testing. I would like to stick to that pattern if possible.

Good to know. Made the necessary changes using fixtures.

**Pull request merged**